src/axom/slic/interface/slic_macros.hpp

```diff
@@ -223,6 +224,14 @@
 
 /// @}
 
+// Use assert when on device
+#elif defined(AXOM_DEBUG) && defined(AXOM_DEVICE_CODE)
+
+#define SLIC_ASSERT( EXP )  assert (EXP)                                    
+#define SLIC_ASSERT_MSG( EXP, msg ) assert (EXP) 
+#define SLIC_CHECK( EXP ) assert (EXP)
+#define SLIC_CHECK_MSG( EXP, msg ) assert (EXP)
```

Comment:
Nice! We are currently not handling `SLIC_ERROR`, `SLIC_ERROR_IF` and `SLIC_WARNING` right? (This can be done in a different PR)

Reply:
Tracked in #178

Reply:
Thanks for adding the issue.